hkmc2DiffTests/src/test/scala/hkmc2/MLsDiffMaker.scala

```diff
@@ -69,13 +75,21 @@ abstract class MLsDiffMaker extends DiffMaker:
       // * Perhaps this should be the default behavior of TraceLogger.
       if doTrace then super.trace(pre, post)(thunk)
       else thunk
+      
+  val rtl = new TraceLogger:
+    override def doTrace = dbgResolving.isSet
+    override def emitDbg(str: String): Unit = output(str)
   
   var curCtx = Elaborator.State.init
+  var curICtx = ImplicitResolver.ICtx.empty
   
+  var prelude = Elaborator.Ctx.empty
   
   override def run(): Unit =
-    if file =/= preludeFile then importFile(preludeFile, verbose = false)
+    // if file =/= preludeFile then 
```

Comment:
Any idea why this is no longer needed? 🤔

Reply:
Ah, this is a workaround that I now realize it is useless. I just reverted it.

Reply:
On a side note, it's interesting that this doesn't cause a problem even running diff-tests on `Prelude.mls`.